### PR TITLE
[plugin.video.mlbbasesloaded] 0.3.0

### PR DESCRIPTION
**Pull request by jakecar**
### Description

* Critical bug fix: the plugin would fail when a blacked out game was selected, resulting in the plugin not switching between games.

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by Tadly** [comment]
You kinda forgot to bump the version number in your `addon.xml`

**Comment by jakecar** [comment]
:)

**Comment by Tadly** [comment]
That's better :)
LGTM

File: plugin.video.mlbbasesloaded/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.mlbbasesloaded" name="MLB BasesLoaded" version="0.2" provider-name="Jake Kramer">
+<addon id="plugin.video.mlbbasesloaded" name="MLB BasesLoaded" version="0.3" provider-name="Jake Kramer">
```
Comment (Razzeee) [inline plugin.video.mlbbasesloaded/addon.xml]: Sorry, but it seems like we missed this the last times, version number should be 0.3.0 so three parts.
http://kodi.wiki/view/Addon.xml#version_attribute

**Pull request merged**